src/globus_cli/commands/flows/run/list.py

```diff
@@ -45,7 +73,10 @@ def list_command(
 
     paginator = Paginator.wrap(flows_client.list_runs)
     run_iterator = PagingWrapper(
-        paginator(filter_flow_id=filter_flow_id).items(),
+        paginator(
+            filter_flow_id=filter_flow_id,
+            filter_roles=",".join(filter_role),
```

Comment:
Similarly to the list_flows case, I might prefer `list(...)` to comma-joining when the SDK already does so.
But actually my favorite option is to write this as
```python
filter_roles=filter_role,  # type: ignore[arg-type]
```

Because it will work today and as soon as we ship an SDK version which fixes the annotation, it will be flagged as an unused ignore by `mypy`.